packages/block-library/src/latest-posts/index.php

```diff
@@ -151,6 +143,24 @@ function render_block_core_latest_posts( $attributes ) {
 
 			$trimmed_excerpt = get_the_excerpt( $post );
 
+			/*
+			 * Adds a "Read more" link with screen reader text.
+			 * [&hellip;] is the default excerpt ending from wp_trim_excerpt() in Core.
+			 */
+			if ( str_ends_with( $trimmed_excerpt, ' [&hellip;]' ) ) {
```

Comment:
@ramonjd Would this be better?

```php
/** This filter is documented in wp-includes/formatting.php */
if ( str_ends_with( $trimmed_excerpt, apply_filters( 'excerpt_more', ' ' . '[&hellip;]' ) ) {
```
[Reference](https://developer.wordpress.org/reference/hooks/excerpt_more/)